### ISSUE TITLE
doesnt do anything

### ISSUE THREAD

**Issue by blezinski** [Shnatsel/furaffinity-dl#3]
running in a terminal window on ubuntu or bash on windows just returns me to the command line without doing anything
if i dont enter any parameters it shows some help stuff so i know i got the path right

**Comment by Shnatsel** [comment]
Could you add "set -x" to the beginning of the script and post the output? That should activate debug mode and point to what exactly goes wrong.

**Comment by blezinski** [comment]
bash$ bash C:/share/fart/trunk/furaffinity-dl gallery/test
+ set -e
+ [ gallery/test =  ]
+ [ gallery/test = -h ]
+ [ gallery/test = --help ]
+ runtime_dir=/.cache/furaffinity-dl
+ mkdir -p /.cache/furaffinity-dl
++ umask u=rwx,g=,o=
++ mktemp /.cache/furaffinity-dl/fa-dl.XXXXXXXXXX
bash$

**Comment by Forst** [comment]
Seems to be related to 39c9676.

**Comment by Shnatsel** [comment]
I've tried it on Windows 10 "bash on ubuntu on windows" in a developer virtual machine (from [here](https://developer.microsoft.com/en-us/windows/downloads/virtual-machines)) build 201612. And it worked perfectly for me. 

I've opened "bash on ubuntu on windows" and entered the following:

```
sudo apt-get install git
git clone https://github.com/Shnatsel/furaffinity-dl.git
```

This downloaded the repository. Then I could run it like this:

```
cd /mnt/c/Users/MyUsername/
~/furaffinity-dl/furaffinity-dl gallery/kodardragon
```
